app/plugins/privileges/static/i18n/de.json

```diff
@@ -0,0 +1,5 @@
+{
+  "NAVIGATION": {
+    "PRIVILEGES": "Privileges"
```

Comment:
Are these the right german/spanish words? Privilege in german is Privilegien, but im not sure if thats the valid word to use in this context. Worth checking